Sources/FoundationEssentials/Data/ContiguousBytes.swift

```diff
@@ -109,3 +124,34 @@ extension Slice : ContiguousBytes where Base : ContiguousBytes {
         }
     }
 }
+
+#if compiler(>=6.2)
+
+//===--- Span Conformances -----------------------------------------===//
+
+@available(FoundationPreview 6.3, *)
+extension RawSpan: ContiguousBytes {
+}
+
+@available(FoundationPreview 6.3, *)
+extension MutableRawSpan: ContiguousBytes {
+}
+
+@available(FoundationPreview 6.3, *)
+extension Span: ContiguousBytes where Element == UInt8 {
+}
+
+@available(FoundationPreview 6.3, *)
+extension MutableSpan: ContiguousBytes where Element == UInt8 {
+}
+
+@available(FoundationPreview 6.3, *)
+@available(macOS 26.0, iOS 26.0, tvOS 26.0, watchOS 26.0, visionOS 26.0, *)
```

Comment:
The double availability attribute seems a bit suspicious here - each declaration should only have a single availability. Is the problem here the macOS back deployment build of the package? If so, we'll need to create a new availability macro (something like `FoundationInlineArray`) that maps to macOS 26-aligned in the package manifest / CMake files

Reply:
This worked out, thank you for the suggestion